soc/arm/nxp_imx/rt/CMakeLists.txt

```diff
@@ -9,9 +9,9 @@ zephyr_sources_ifdef(CONFIG_SOC_SERIES_IMX_RT10XX soc_rt10xx.c)
 zephyr_linker_sources_ifdef(CONFIG_NXP_IMX_RT_BOOT_HEADER
   ROM_START SORT_KEY 0 boot_header.ld)
 
-# Enable custom OCRAM noncacheable region
+# Add custom mpu regions
+zephyr_sources(mpu_regions.c)
```

Comment:
maybe something should go in the release notes about this change?

Reply:
Sure- I'll highlight this in the next update of this PR